.github/workflows/nintendo.yaml

```diff
@@ -8,7 +8,7 @@ on:
 
 jobs:
   _3ds:
-    name: Nintendo 3DS
+    name: Nintendo 3DS 3dsx
```

Comment:
Should probably be `Nintendo 3DS (3dsx)` if the syntax allows for it.

Reply:
Done [4efabe6](https://github.com/angband/angband/pull/5376/commits/4efabe63c42fda1dc5779b6d6d79651a115c5282)